addon/components/base/bs-form/element.js

```diff
@@ -983,6 +983,7 @@ export default FormGroup.extend({
       this.set('showValidationOn', ['focusOut']);
     }
     if (!isBlank(this.get('property'))) {
+      assert('You cannot set both property and value on a form element', isBlank(this.get('value')));
```

Comment:
This would allow having `property` but `value` being set to an empty string, which is probably a very common initial value. So the assertion wouldn't trigger in that case. Checking for a `null` value (which is its default value) might be better?

```suggestion
      assert('You cannot set both property and value on a form element', this.get('value') === null);
```

Reply:
Oh that's a good point about initial values- a null check sounds better to me too, thanks!! 😄